samples/net/sockets/big_http_download/src/big_http_download.c

```diff
@@ -33,6 +33,11 @@
 
 #endif
 
+#define KiB	*1024
+#define MiB	*1024 KiB
+#define Bytes2KiB(Bytes)	(((float)Bytes)/(1.0f KiB))
+#define Bytes2MiB(Bytes)	(((float)Bytes)/(1.0f MiB))
+
```

Comment:
from @pfalcon in #26719 : I'm not sure about usage of floating-point arithmetic. This would be one cause of stack usage growth you faced. And it would grow binary size, and potentially make it not fit on smaller devices, which would be a loss of functionality.

Reply:
Will be changed to int arithmetics.